app/jobs/regular/create_ai_reply.rb

```diff
@@ -15,7 +15,7 @@ def execute(args)
 
         bot = DiscourseAi::Personas::Bot.as(bot_user, persona: persona.new)
 
-        DiscourseAi::AiBot::Playground.new(bot).reply_to(post)
+        DiscourseAi::AiBot::Playground.new(bot).reply_to(post, feature_name: "bot")
```

Comment:
Would it make sense to pass this from the job caller?

Reply:
hmmm maybe ... this job though is dedicated to bot.